Capstrone/assessmentportal/src/User/Students.jsx

```diff
@@ -16,15 +17,16 @@ function Student() {
   useEffect(() => {
     getStudents();
   }, []);
-  const getStudents= async () => {
-  
-      // const response = await axios.get('http://localhost:6002/students');
-      UserApi.getAllStudents().then(response=>{
+  const getStudents = async () => {
+
+    // const response = await axios.get('http://localhost:6002/students');
```

Comment:
Remove commented codes and consoles everywhere

Reply:
Resolved